test/support/ssl/esme.ex

```diff
@@ -13,12 +13,7 @@ defmodule Support.SSL.ESME do
       @host,
       port,
       {__MODULE__, %{pid: self()}},
-      transport: :ranch_ssl,
-      transport_opts: [
```

Comment:
Why do we need to remove this?

Reply:
It's not used, transport_opts is not used, neither is it even mentioned in the docs:

https://github.com/funbox/smppex/blob/master/lib/smppex/esme.ex#L84-L106

Reply:
Oh, that's interesting 🤔